crates/iota/src/iota_commands.rs

```diff
@@ -818,12 +818,23 @@ async fn start(
         start_test_indexer(
             Some(pg_address.clone()),
             fullnode_url.clone(),
-            ReaderWriterConfig::reader_mode(indexer_address.to_string()),
-            data_ingestion_dir,
+            IndexerTypeConfig::reader_mode(indexer_address.to_string()),
+            data_ingestion_dir.clone(),
             None,
         )
         .await;
         info!("Indexer in reader mode started");
+
+        // Start in analytical worker mode
+        start_test_indexer(
```

Comment:
I know there are already 2 workers by why are we using something from `test_utils` in the CLI?

Reply:
I'm not sure about this, but I think it is because the local network is also for testing and by the db just gets reset after shutting it down
If we would use the iota-indexer start methods directly we would have to add the things in `start_test_indexer()` additionally to get the db cleaned up afterwards

Reply:
Fair enough